.github/workflows/ansible-lint.yml

```diff
@@ -1,46 +1,23 @@
 ---
-name: Ansible-lint
+name: Ansible Lint
 
-on:
+on:  # yamllint disable-line rule:truthy
```

Comment:
This .yamllint config item takes care of the 'on:', the disable-line is not needed.

 `check-keys: false`

Reply:
fixed.